core/src/storage/cache.py

```diff
@@ -227,6 +227,19 @@ def set(key: int, value: bytes) -> None:
     _SESSIONS[_active_session_idx].set(key, value)
 
 
+def set_int(key: int, value: int) -> None:
+    if key & _SESSIONLESS_FLAG:
+        length = _SESSIONLESS_CACHE.fields[key ^ _SESSIONLESS_FLAG]
+    elif _active_session_idx is None:
+        raise InvalidSessionError
+    else:
+        length = _SESSIONS[_active_session_idx].fields[key]
+
+    encoded = value.to_bytes(length, "big")
+    utils.ensure(int.from_bytes(encoded, "big") == value)
```

Comment:
I'm wondering whether this shouldn't be an `assert` instead of `ensure`. There seems to be very little value in this RSODing randomly in production?